firebase.json

```diff
@@ -1062,6 +1062,21 @@
         "destination": "/tools/faq#pub",
         "type": 301
       },
+      {
```

Comment:
Would be better to redirect to something less confusing. Is there a source document that explains why we're done with transformers?

Reply:
We could possibly link to https://v1-dartlang-org.firebaseapp.com/tools/pub/assets-and-transformers and put a big note on there that transformers won't be supported in Dart 2.

There's also https://github.com/dart-lang/build/blob/master/build/README.md, but maybe the assets & transformers note should point there.

Reply:
I've created a new page, [/tools/pub/obsolete](https://kw-www-dartlang-1.firebaseapp.com/tools/pub/obsolete) that has a bit of explanation and a lot of links. What do you think?

Reply:
(It doesn't, however, explain _why_ we dropped support for transformers. Do you think that's necessary?)

Reply:
Awesome!